tools/docker/dev_container.Dockerfile

```diff
@@ -1,11 +1,9 @@
 #syntax=docker/dockerfile:1.1.5-experimental
-FROM gcr.io/tensorflow-testing/nosla-cuda11.2-cudnn8.1-ubuntu18.04-manylinux2010-multipython as dev_container_cpu
+ARG PY_VERSION
+FROM tensorflow/build:latest-python$PY_VERSION as dev_container_cpu
```

Comment:
I think that we could need to add comment here that this is a GPU only image with a reference to https://github.com/tensorflow/build/pull/47 or any related ticket that you like (on TF or Addons).

Reply:
Done.